routes/licenses.test.js

```diff
@@ -46,7 +49,35 @@ describe('license routes', () => {
     });
   });
 
-  describe('GET /license', () => {
+  describe('GET /licenses/compatible/{license}', () => {
+    function options() {
+      return { url: `/licenses/compatible/CC+BY-SA+3.0`, method: 'GET' };
```

Comment:
Not sure if really necessary, but we could also add a negative example of a "wrongly" _(or at least differently than what the endpoint expects)_ encoded license string 🤔

Reply:
Generally I'd agree that testing error-cases makes sense. I'm having a hard time to come up with an example of _wrong_ license name input, that causes an error in this route handler's function.
What we want to test here is wether any error that is thrown from `decodeURIComponent` is handled properly. I cannot trigger such an error though, because any invalid uri encoding seems to be handled by hapi before the handler function is even called.
I could add a test that asserts that the route returns 400 for invalid uri input, but I think that this is not useful as this only tests hapi's behaviour and there is no explicit error handling implemented yet.

Reply:
Would a license-string like this already fail?
```javascript
'CC==BY==SA++3.0'
```

Valid point about wrong URLs and not testing `decodeURICompontent` 👍

Reply:
This ☝️ string would not fail as it is properly encoded, and I mocked out the service that the parameter is passed to. It should result in either an empty response or an error response. We should take this into consideration when we implement error handling for this route!